### PR TITLE
Beta3.2

### PR DESCRIPTION
**Pull request by timoj**
- Python 3.9 support
- Renamed model to module
- Use aiojobs for background tasks

### CODE REVIEW

File: lib/series/seriesmanager.py [hunk 1]
```diff
@@ -234,7 +239,9 @@ async def read_from_disk(cls):
             series_data = data.get('series')
             if series_data is not None:
                 for s in series_data:
-                    cls._series[s.get('name')] = Series.from_dict(s)
+                    await cls._add_series(s)
+                    # cls._series[s.get('name')] = Series.from_dict(s)
+                    # cls._series[s.get('name')]
```
Comment (robbm1) [inline lib/series/seriesmanager.py:244]: Why are these lines commented out?

File: server.py [hunk 2]
```diff
@@ -322,11 +330,10 @@ def start_server(self):
...
             logging.info('Socket.io API enabled')
             self.sio = socketio.AsyncServer(async_mode='aiohttp',
-                                            ping_timeout=60,
-                                            ping_interval=25,
-                                            cookie=None,
-                                            cors_allowed_origins='*',
-                                            logger=False)
+                                            # ping_timeout=60,
+                                            # ping_interval=25,
+                                            # cookie=None,
```
Comment (robbm1) [inline server.py:335]: Why are these lines commented out?

File: lib/series/seriesmanager.py (hunk 1 above)
Comment (timoj) [reply lib/series/seriesmanager.py:244]: Short reminders for myself. I will clean it up before releasing a new version

File: server.py (hunk 2 above)
Comment (timoj) [reply server.py:335]: Short reminders for myself. I will clean it up before releasing a new version

**Review by robbm1 — APPROVED** [review]

**Pull request merged**